src/main/java/frc/robot/subsystems/ArmPivot.java

```diff
@@ -27,47 +26,10 @@
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
 import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
-import frc.robot.Hardware;
+import frc.robot.Constants.ArmPivotConstants;
 import java.util.function.Supplier;
 
 public class ArmPivot extends SubsystemBase {
-  // Presets
-  private final double ARMPIVOT_KP = 38.5; // previously 50
-  private final double ARMPIVOT_KI = 0;
-  private final double ARMPIVOT_KD = 0;
-  private final double ARMPIVOT_KS = 0.1;
-  private final double ARMPIVOT_KV = 0.69;
-  private final double ARMPIVOT_KG = 0.18;
-  private final double ARMPIVOT_KA = 0.0;
-  // Preset positions for Arm with Coral
-  public static final double CORAL_PRESET_L1 = 0;
-  public static final double CORAL_PRESET_L2 = 0.13;
-  public static final double CORAL_PRESET_L3 = 0.13;
-  public static final double CORAL_PRESET_L4 = 0.0;
-  public static final double CORAL_PRESET_PRE_L4 = 1.0 / 16.0;
-  public static final double CORAL_PRESET_STOWED = 0.125;
-  public static final double CORAL_PRESET_OUT = 0;
-  public static final double CORAL_PRESET_UP = 0.25; // Pointing directly upwards
-  public static final double CORAL_PRESET_DOWN = -0.25;
-  // Preset positions for Arm with Algae
-  public static final double CORAL_POST_SCORE = -0.15;
-  public static final double CORAL_QUICK_INTAKE = -0.07;
-  public static final double ALGAE_REMOVE_PREPOS = 0;
-  public static final double ALGAE_REMOVE = 0;
-  public static final double ALGAE_FLING = -0.08;
-  public static final double ALGAE_STOWED = -0.05;
-  public static final double ALGAE_PROCESSOR_SCORE = -0.05;
-  public static final double ALGAE_GROUND_INTAKE = -0.085;
-  public static final double ALGAE_NET_SCORE = 0.175; // untested - old value was 0.18
-
```

Comment:
I'm generally not a fan of splitting code across multiple files for a logical unit (Arm Pivot in this case) since it means always having two files open when working on code related to that component